app/i18n/en/conf.php

```diff
@@ -120,20 +120,33 @@
 			'feeds' => 'Display by feed',
 			'order' => 'Sort by date',
 			'search' => 'Expression',
+			'shareOpml' => 'Enable sharing by OPML of corresponding categories and feeds',
+			'shareRss' => 'Enable sharing by HTML &amp; RSS',
 			'state' => 'State',
 			'tags' => 'Display by label',
 			'type' => 'Type',
 		),
 		'get_all' => 'Display all articles',
+		'get_all_labels' => 'Display articles with any label',
```

Comment:
Is it obsolete? I cannot find any place where it is used

Reply:
Strange, I cannot find all this "get_...." labels used somewhere....
@Alkarex Could you help me please?

Reply:
It is because they are used by string concatenation (which I would like to get rid of precisely because it makes validation and maintenance quite challenging). And there is another layer of property with another name for the getter, which makes it even more difficult to search:

https://github.com/FreshRSS/FreshRSS/blob/0ffcf41f9371b9321856d2c709cf7bcb4f7c3c60/app/views/configure/queries.phtml#L57